nlp/Dockerfile

```diff
@@ -4,13 +4,48 @@ FROM birdhouse/pavics-jupyter-base:0.2.1
 USER root
 
 COPY environment.yml /environment.yml
+COPY /nl2query ./nl2query 
+COPY notebook_config.yml /notebook_config.yml
+
 
 # update env "birdy"
 # use umask 0000 so that package files for the updated environment are usable by the user for the jupyter-conda-extension
 RUN umask 0000 && conda env update -f /environment.yml
 
-COPY notebook_config.yml /notebook_config.yml
 
-# specify user because of problem running start-notebook.sh when being root
-USER jenkins
 
+# Installing java opjdk
+RUN conda install -c anaconda openjdk
```

Comment:
why install `anaconda`, since we already have its lighter version, `conda`? `anaconda` will install hundreds of packages (including `conda`), so it will make the image big

Reply:
Would it be better if I was able to move the `openjdk` and the environment Variable `LANG=C.UTF-8`  into `environment.yml` like this: 
```bash
channels:
  - conda-forge
dependencies:
  - openjdk
variables:
  LANG=C.UTF-8
```
or should we keep it in the dockerfile ?

Reply:
Moving `openjdk` to the conda yml is good, but you can keep the utf-8 param as Dockerfile env var.